### PR TITLE
Added Deep.BI - Predictive Analytics & AI Platform

### PR DESCRIPTION
**Pull request by jmalcone546**
Hi there, I'd like to request to add Deep.BI (https://www.crunchbase.com/organization/deep-bi) to the powered-by list. 

I hope everything is correct, please feel free to reach out for any clarifications, and thank you in advance!

### CODE REVIEW

File: source/powered-by-nifi.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ containerEnabled: true
 
 # Powered by Apache {{< project-label >}}
 
+## [Deep.BI](https://www.deep.bi)
+
+Deep.BI helps enterprises leverage next-gen data & AI pipelines to process hundreds of thousands of real-time events per second, powered by Apache Druid & Apache Flink. Our platform and solutions leverage NiFi as a core component in building robust cybersecurity solutions for critical applications and as a core component of our data processing architecture.
```
Comment (exceptionfactory) [inline source/powered-by-nifi.md:10]:
```suggestion
Deep.BI helps enterprises leverage next generation data and AI pipelines to process hundreds of thousands of real-time events per second, powered by Apache Druid and Apache Flink. The Deep.BI data processing platform leverages Apache NiFi to build robust cybersecurity solutions for critical applications.
```

**Review by exceptionfactory — COMMENTED** [review]
Thanks for requesting the addition to the lists of companies powered by NiFi @jmalcone546!

I suggested some minor wording adjustments, can you let me know if you agree with those changes?

As your GitHub profile is new and does not include any reference to Deep.BI, can you provide some type of confirmation that you represent and that the company has approved this addition? Feel free to contact the Apache NiFi Project Management Committee at private@nifi.apache.org if you have questions.

Thanks!

File: source/powered-by-nifi.md (hunk 1 above)
Comment (jmalcone546) [reply source/powered-by-nifi.md:10]: Hi @exceptionfactory, thank you very much for your reply and suggestions, I think they work really well! I'll confirm it with the team to be fully sure.

I totally understand your concern, I probably should've asked one of our team members to do it from their active GitHub accounts.

I'll send an email to officially confirm this from our company email domain, I think that'll be the easiest, thank you!

Comment (exceptionfactory) [reply source/powered-by-nifi.md:10]: Thanks for the reply @jmalcone546!

Sending an email from your company domain sounds good. I will plan on merging the addition once I receive the email confirmation.

**Review by exceptionfactory — APPROVED** [review]
Merging based on reference to Apache NiFi along with other Apache products at the following URL:

https://www.deep.bi/solutions/devops-big-data-engineering-ai-solutions

Thanks @jmalcone546!

**Pull request closed without merging**